### PR TITLE
Rename emhttpsrc to webfetchsrc

### PR DESCRIPTION
**Pull request by aslobodeniuk**
Issue: RDI_2054

### CODE REVIEW

File: gst.wasm/subprojects/gst-plugins-web/gst/webfetchsrc/gstwebfetchsrc.c [hunk 1]
```diff
@@ -57,36 +57,36 @@ typedef struct _GstEmHttpSrc
   gsize download_offset;
   gsize download_end;
   gsize resource_size;
-} GstEmHttpSrc;
+} WebFetchSrc;
 
 G_DECLARE_FINAL_TYPE (
-    GstEmHttpSrc, gst_em_http_src, GST, EM_HTTP_SRC, GstPushSrc)
-G_DEFINE_TYPE_WITH_CODE (GstEmHttpSrc, gst_em_http_src, GST_TYPE_PUSH_SRC,
+    WebFetchSrc, gst_webfetch_src, GST, WEBFETCH_SRC, GstPushSrc)
```
Comment (cfoch) [inline gst.wasm/subprojects/gst-plugins-web/gst/webfetchsrc/gstwebfetchsrc.c:63]: Use gst_web_fetch_src. We do something similar in gst_web_canvas_sink and gst_web_canvas_src.

File: gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c [hunk 2]
```diff
@@ -62,13 +62,13 @@ init_pipeline ()
   GST_DEBUG ("Init pipeline");
 
   GST_PLUGIN_STATIC_DECLARE (coreelements);
-  GST_PLUGIN_STATIC_DECLARE (emhttpsrc);
+  GST_PLUGIN_STATIC_DECLARE (webfetchsrc);
```
Comment (cfoch) [inline gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c:65]: We should ideally include webfetchsrc in the web plugin.

**Review by cfoch — CHANGES_REQUESTED** [review]

File: gst.wasm/subprojects/gst-plugins-web/gst/webfetchsrc/gstwebfetchsrc.c (hunk 1 above)
Comment (aslobodeniuk) [reply gst.wasm/subprojects/gst-plugins-web/gst/webfetchsrc/gstwebfetchsrc.c]: fixed

File: gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c (hunk 2 above)
Comment (aslobodeniuk) [reply gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c:65]: I agree, but I don't take these desicions, IMHO that should be planned by the designers of the project @turran @rgonzalezfluendo

**Review by cfoch — APPROVED** [review]
Thanks. Remember to wait for https://fluendo.atlassian.net/browse/RDI-2054 before merge.

File: gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c (hunk 2 above)
Comment (turran) [reply gst.wasm/subprojects/samples/webfetchsrc/webfetchsrc-example.c:65]: It is agreed this way, yes. Only one plugin for web related plugins

**Pull request merged**